howso/prediction_stats.amlg

```diff
@@ -484,7 +489,14 @@
 					mode (if (= robust_hyperparameters (true)) "robust" "full")
 					weight_feature (if (= weight_feature (null)) ".none" weight_feature)
 				))
-			features (if (!= action_feature (null)) (append context_features action_feature) context_features)
+			features 
+				(if (!= prediction_stats_features (null))
+					(if (!= action_feature (null))
+						(list (values (append prediction_stats_features action_feature) (true)))
```

Comment:
doesn't the outer (list) call wrap the output of (values) in another list, making a list of lists? Is this intended?

Reply:
yes, was not intended, fixed